src/processes/registerv2-process.js

```diff
@@ -83,7 +83,7 @@ const registerProcess = async (job, done) => {
     await ProcessHelper.followMainFeedFollowing(userId, follows);
     console.log('::: Start addUserToTopicChannel :::');
     await ProcessHelper.addUserToTopicChannel(userId, topicNames);
-    await syncFeedPerUserProcess(userId);
+    await syncFeedPerUserProcess(userId, 'registerUser');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change seems to be adding a new parameter 'registerUser' to the `syncFeedPerUserProcess` function call. However, it's not clear what this string represents. It would be better to use a constant or an enumeration to represent this value, which can improve readability and maintainability.

```diff
-    await syncFeedPerUserProcess(userId, 'registerUser');
+    const USER_REGISTRATION_SOURCE = 'registerUser';
+    await syncFeedPerUserProcess(userId, USER_REGISTRATION_SOURCE);
```
This way, if the value needs to be changed in the future, it only needs to be updated in one place. Also, using a constant makes the code more self-documenting.


<!-- This is an auto-generated comment by OSS CodeRabbit -->